cumulus_library/studies/core/core_templates/documentreference.sql.jinja

```diff
@@ -60,7 +60,7 @@ temp_encounters AS (
          unnest(context.encounter) AS context_encounter (encounter) --noqa
 
 {% else %}
-        '' AS encounter_ref
+        tdr.id AS encounter_ref
```

Comment:
I found that athena will error out if you try to make a zero-length varchar? :shrug: So I switched to grabbing the type from the base table

Reply:
could you get to the same behavior with an explicit cast? it might be a little more explicit that way.

Reply:
Yup, that works - will switch.